cmd/deploy.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/docker/libcompose/docker"
+	"github.com/docker/libcompose/docker/ctx"
+	"github.com/docker/libcompose/project"
+	"github.com/spf13/cobra"
+)
+
+// deployCmd represents the up command
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "Trigger an image deployment for all shipments and containers defined in compose files",
+	Long:  "Note that the deploy command is a subset of the up command without updates for environment variables, replicas, barge info, etc.",
+	Run:   deploy,
+}
+
+var shipmentBuildToken string
+
+func init() {
+	deployCmd.PersistentFlags().StringVarP(&shipmentBuildToken, "token", "t", "", "the shipment build token to use")
```

Comment:
Yes, was going to until I realized some users are deploying to multiple environments. Thought this might be easier than changing the env var.

```
- harbor-compose deploy --token ${BUILD_TOKEN_DEV}
- harbor-compose deploy --token ${BUILD_TOKEN_QA}
```